### ISSUE TITLE
When signing ICLA wrong contributor name is added to the CLA agreement

### ISSUE THREAD

**Issue by pranab-bajpai** [communitybridge/easycla#2499]
Here are the steps:
- [ ] Create a CLA Group as user pranabbajpai, associate GitHub repositories for the project with this CLA Group
- [ ] Now open an incognito window and login to GitHub as user test.pm.dec2020@gmail.com (password : LinuxFoundation2020). Browse to any repo that is part of CLA Group in previous step, make change to readme file and get blocked on a PR.
![image](https://user-images.githubusercontent.com/57362064/105410394-7d2bbc00-5bf7-11eb-90a9-12a874ea2889.png)
- [ ] Proceed to sign an ICLA. You will notice that the ICLA has a different user's name on it, it should have had test.pm.dec2020@gmail.com
- [ ] 
![image](https://user-images.githubusercontent.com/57362064/105410588-c5e37500-5bf7-11eb-9598-878fc92a5609.png)

**Comment by vinod-kadam** [comment]
It was marked as low priority as it was not seen by others. So have marked it as Post-EasyCLAV2Release.

**Comment by dealako** [comment]
Won't fix. Out of scope of current product requirements/features.
